src/fileformats/ocd_file_export.cpp

```diff
@@ -1814,9 +1814,9 @@ void OcdFileExport::exportTextSymbol(OcdFile<Format>& file, const TextSymbol* te
 {
 	auto symbol_number = symbol_numbers.at(text_symbol);
 	
-	text_format_mapping.push_back({ text_symbol, TextObject::AlignLeft, 0, symbol_number });
-	text_format_mapping.push_back({ text_symbol, TextObject::AlignHCenter, 0, symbol_number });
-	text_format_mapping.push_back({ text_symbol, TextObject::AlignRight, 0, symbol_number });
+	for (auto alignment : {TextObject::AlignLeft, TextObject::AlignHCenter, TextObject::AlignRight})
+		text_format_mapping.push_back({ text_symbol, alignment, 0, symbol_number });
+
```

Comment:
I'm unsure if this change is really beneficial. With the extra line, it is really harder to understand that `text_format_mapping.end() - 3` below refers to the first entry which was added here.

Reply:
The pull request comment has the "take them or leave them" suffix. I know that our views on coding do not align in many areas.

Here I made the change to express the fact that it is only the alignment changing in the format. Other parameters are constant. The previous form requires attentive reading and comparison of the three lines for differences to come to the same conclusion. Try that on Friday afternoon after a week of coding and maybe your stomach will tell you which form is more digestible.

Ad the `text_format` iterator, yes, it's a somewhat strange element here. Its initialization nicely aligns with its nature but I didn't tackle this bit.